hello.rb

```diff
@@ -0,0 +1,63 @@
+def check(next_x, next_y, sol)	 
+	sol.each do |i|
+	end	
+	if( next_x >= 0 && next_x < 5 && next_y >= 0 && next_y < 5 && sol[next_x][next_y] == -1)
+		return true
+	else
+		return nil
+	end		
+end	
+
+def display(solArr)
+	solArr.each do |i|
+		puts "#{i}"
+	end	
+end	
+
+def possibleMove(x,y,totalMove,solArr,xMove,yMove)
+
+	tableLen = solArr.length
+	
+	if (totalMove == tableLen * tableLen + 1)
```

Comment:
@kamal-webonise  why explicitly returned true as I told that everything except false and nil is treated as true.